.github/workflows/docker.yml

```diff
@@ -1,6 +1,9 @@
 name: docker
 
 on:
```

Comment:
The release job had do this.

Reply:
yeah. This PR will solve the issue which we created a version branch and push new commits to this branch before releasing.

Reply:
It's triggered after the release branch is created and before the released tag is created. During this time, if new commits pushed to the released branch, the docker image should be updated in time.